adaptdl/adaptdl/torch/adascale.py

```diff
@@ -227,14 +226,21 @@ def _final_callback(self):
             self._async_op.wait()
 
         grads = []
+        if self._mp_scaler is not None:
+            mixed_precision_scale = self._mp_scaler.get_scale()
+        else:
+            mixed_precision_scale = 1.0
         for group in self._optimizer.param_groups:
             grads.append([])
             for param in group["params"]:
                 if param.grad is None:
                     grads[-1].append(None)
                     continue
-                param.grad.div_(self._accum_count)
-                grads[-1].append(param.grad.detach())
+                param.grad.div_(self._accumulation_steps)
```

Comment:
dividing by the wrong variable?

Reply:
Yea, I missed that change during the rebase